### PR TITLE
Feature/update helm

### PR DESCRIPTION
**Pull request by dbampalikis**
The pull requests main goal is to update the helm charts for the service, in order to work with the setup of the new cluster. Specifically it includes:
- Updates in the go service to support the new variable names
- Updates in the helm charts to support the new secrets (such as the `crypt4gh` public key) and changes in the names (such as the `JWT` key)
- Updates in the helm charts to support the new way of requesting certificates
- Workflows for manual and automatic release of the charts

### CODE REVIEW

**Review by norling — APPROVED** [review]
I'm no expert helmologist but as far as I can tell this seems very nice! :)

File: README.md [hunk 1]
```diff
@@ -45,9 +45,24 @@ go run .
...
+```sh
+docker build -t harbor.nbis.se/uppmax/integration .
+docker push harbor.nbis.se/uppmax/integration
+```
+Create a secret using
+kubectl -n lega create secret generic something --from-literal=jwt.key=<some_key>
+```
+Populate the `values.yaml` file with the correct values and then install using the local copy of the helm charts with
+```sh
+helm install --namespace lega uppmax .
```
Comment (jbygdell) [inline README.md:67]:
```suggestion
helm install --namespace lega uppmax charts/uppmax-integration
```

**Review by jbygdell — APPROVED** [review]
Small change in the README but otherwise it LGTM

**Review by shreyasshivakumara — APPROVED** [review]
Looks good!

**Pull request merged**